tests/fixtures/testrepo/upstream/package_manifest.yaml

```diff
@@ -10,6 +10,8 @@ packages:
     koji_tags:
       - name: obaltest-nightly-rhel7
         dist: '.el7'
+      - name: obaltest-nightly-el8
+        dist: '.el8'
```

Comment:
this made some other tests fail as they have not been updated to check for both el7 and el8 things:


```
FAILED tests/test_functional.py::test_obal_check_upstream_hello
FAILED tests/test_functional.py::test_obal_release_upstream_hello
FAILED tests/test_functional.py::test_obal_release_upstream_hello_nowait
FAILED tests/test_functional.py::test_obal_release_upstream_hello_nowaitrepo
```